packages/pwc-utils/src/polyfills/index.js

```diff
@@ -0,0 +1,24 @@
+// NOTE: `Array.from()` and `Object.assign()` and event constructor are covered by `@webcomponents/webcomponents-platform`
+// NOTE: The need of iterator (`for` ... `of` loop) support is not revealed by Storybook,
+// but by testing `carbon-custom-elements-with-polyfills.js` stand-alone
```

Comment:
I'm wondering if we truly need all these polyfills? The comments suggest some polyfills are required for Carbon. 

Would be good to clean this up if possible?